test/acceptance/tests/sync/sync_catalog_test.go

```diff
@@ -0,0 +1,124 @@
+package sync
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul-helm/test/acceptance/framework"
+	"github.com/hashicorp/consul-helm/test/acceptance/helpers"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes"
+)
+
+func TestSyncCatalog(t *testing.T) {
+	cases := []struct {
+		name       string
+		helmValues map[string]string
+	}{
+		{
+			"Default installation",
+			map[string]string{
+				"syncCatalog.enabled": "true",
+			},
+		},
+		{
+			"Secure installation (with TLS and ACLs enabled)",
+			map[string]string{
+				"syncCatalog.enabled":          "true",
+				"global.tls.enabled":           "true",
+				"global.acls.manageSystemACLs": "true",
+			},
+		},
+	}
+
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+			env := suite.Environment()
+
+			helmValues := map[string]string{
+				"syncCatalog.enabled": "true",
+			}
+
+			releaseName := helpers.RandomName()
+			consulCluster := framework.NewHelmCluster(t, helmValues, env.DefaultContext(t), suite.Config(), releaseName)
+
+			consulCluster.Create(t)
+
+			t.Logf("creating a test service and pod called %s", releaseName)
+			createTestService(t, env.DefaultContext(t).KubernetesClient(t), releaseName)
+
+			consulClient := consulCluster.SetupConsulClient(t, false)
```

Comment:
Good catch! 🤦‍♀️ that is because I forgot to provide helmValues from the test case. Instead, I'm using the same ones (that are set up a few lines above) in both test scenarios. Fixed!